gatsby/config-options/gatsby-plugin-feed-options.tsx

```diff
@@ -106,6 +106,11 @@ export const rssRenderOptions: Partial<Options> = {
       }
     },
     [BLOCKS.EMBEDDED_ASSET]: (node) => {
+      if (!node.data.target.file) {
+        console.log(JSON.stringify(node));
```

Comment:
well maybe that's a little bit too debugging? Could you log a message and maybe mention some other details.
Or alternative to get things going: Remove the console .log
Or alternative 2: I know that you're working on this topic anyway so let it be  😅

Reply:
Thank you for your review, I added a fix for the original problem which should be a better way of fixing the build